### PR TITLE
style: repeated-global (FURB154)

### PR DESCRIPTION
**Pull request by echoix**
Ruff rule: https://docs.astral.sh/ruff/rules/repeated-global/

### CODE REVIEW

**Review by ninsbl — APPROVED** [review]

**Comment by ninsbl** [comment]
Hm... I see the one line in _python/grass/temporal/core.py_ gets too long...

**Comment by echoix** [comment]
I thought I applied black one last time... Let's try again

**Comment by ninsbl** [comment]
See: https://github.com/OSGeo/grass/actions/runs/9923095792

**Comment by echoix** [comment]
> See: https://github.com/OSGeo/grass/actions/runs/9923095792

With the conflicts solved, there aren't anything else to fix now

**Comment by echoix** [comment]
Black reverts it... So maybe suggest a revert with a `# noqa: FURB154` and apply it, have to go

**Comment by ninsbl** [comment]
> Black reverts it... So maybe suggest a revert with a `# noqa: FURB154` and apply it, have to go

Not sure what you mean, but excluding those two files from the rule sounds reasonable...

File: python/grass/temporal/core.py [hunk 1]
```diff
@@ -562,17 +560,7 @@ def init(raise_fatal_error=False, skip_db_version_check=False):
...
-    global tgis_database_string
-    global tgis_dbmi_paramstyle
-    global tgis_db_version
-    global raise_on_error
-    global enable_mapset_check
-    global enable_timestamp_write
-    global current_mapset
-    global current_location
-    global current_gisdbase
+    global tgis_backend, tgis_database, tgis_database_string, tgis_dbmi_paramstyle, tgis_db_version, raise_on_error, enable_mapset_check, enable_timestamp_write, current_mapset, current_location, current_gisdbase
```
Comment (echoix) [inline python/grass/temporal/core.py:563]:
```suggestion
    global tgis_backend, tgis_database, tgis_database_string  # noqa: FURB154
    global tgis_dbmi_paramstyle, tgis_db_version  # noqa: FURB154
    global raise_on_error  # noqa: FURB154
    global enable_mapset_check, enable_timestamp_write  # noqa: FURB154
    global current_mapset, current_location, current_gisdbase  # noqa: FURB154

```

**Review by ninsbl — APPROVED** [review]

**Pull request merged**